guide/start/policies.md

```diff
@@ -9,169 +9,423 @@ children:
 - { section: "Managing with Policies", title_in_menu: "Policies"  } 
 ---
 
+## A Clustered Example
 
+We'll now look at a more complex example that better shows the capabilities of Brooklyn, including
+inlife management of a clustered application.
+
+Below is the annotated blueprint.
+
+Copy the blueprint into a text file, `mycluster.yaml`, in your workspace, but *before* you create 
+an application with it, review and/or change the the location where the application will be 
```

Comment:
I would put the break in later "Below is an annotated blueprint, copy this blueprint into a text file, `mycluster.yaml`, in your workspace. Before..."